main.go

```diff
@@ -20,8 +20,8 @@ import "github.com/atomist-skills/go-skill"
 
 func main() {
 	skill.Start(skill.Handlers{
-		"on_push": TransactCommitSignature,
+		"on_push":             TransactCommitSignature,
```

Comment:
These keys should probably be defined by the libarary as contans, or perhaps actual struct fields?

Reply:
Those keys are defined by the actual skill as those are the names of the subscriptions. Not sure this can be in the library or an be struct fields.

Reply:
Ah, ok, I thought that could the case... 👍